ui/widget.js

```diff
@@ -367,16 +369,33 @@ $.Widget.prototype = {
 		return this;
 	},
 	_setOption: function( key, value ) {
+		var classKey, elements;
+		if ( key === "classes" ) {
+			for ( classKey in value ) {
+				if ( value[ classKey ] !== this.options.classes[ classKey ] ) {
+					if ( this.classObject[ classKey ] && this.classObject[ classKey ].length ) {
+						elements = $( this.classObject[ classKey ].get() );
```

Comment:
This is the third reference to `this.classObject[ classKey ]`, let's put it in a var.